src/frontend/scss/_main.scss

```diff
@@ -3,8 +3,16 @@
 @import './tools/rem';
 @import './tools/colors';
 
-// Our variables must be imported before Bootstrap code so they can override it
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+
 @import './settings/fonts';
+@import './settings/colors';
+@import './settings/gradients';
+@import './settings/color_schemes';
+@import './settings/theme';
```

Comment:
Import those files from `_main.scss` introduces a breaking change for people using `richie-site-factory`. I think we could continue to import them from `_variables.scss`.

Reply:
Ok.